elegy/losses/binary_crossentropy.py

```diff
@@ -0,0 +1,92 @@
+from elegy import types
+import typing as tp
+import jax
+import jax.numpy as jnp
+from elegy import utils
+from elegy.losses.loss import Loss, Reduction
+
+
+def binary_cross_entropy(
+    y_true: jnp.ndarray, y_pred: jnp.ndarray, from_logits: bool = False
+) -> jnp.ndarray:
+    if from_logits:
+        y_pred = jax.nn.log_sigmoid(y_pred)
```

Comment:
@sebasarango1180 We tried this right? I think its not the correct formula.

Reply:
@cgarciae I'm validating this. And sure, I will check the testing.